android/gradle.properties

```
@@ -1,4 +1,4 @@
-PassageReactNative_kotlinVersion=1.7.0
+PassageReactNative_kotlinVersion=1.8.10
```

Should this be `1.8.1`?
```suggestion
PassageReactNative_kotlinVersion=1.8.1
```